Error when installed on fresh CRA

When trying to add this on a fresh CRA (via `yarn add blocks-ui`) one gets the following error:

```
./node_modules/blocks-ui/dist/index.js
Module not found: Can't resolve '@blocks/blocks/src' in '.../node_modules/blocks-ui/dist'
```

I'm still getting up to speed on the crazy world of npm packaging, so unfortunately I haven't been able to dig much deeper than that.

--- a/packages/blocks-ui/src/index.js
+++ b/packages/blocks-ui/src/index.js
@@ -4,7 +4,7 @@ import { DragDropContext, Droppable, Draggable } from 'react-beautiful-dnd'
 import { Styled, jsx } from 'theme-ui'
 import * as presets from '@theme-ui/presets'
 
-import * as DEFAULT_BLOCKS from '@blocks/blocks/src'
+import * as DEFAULT_BLOCKS from '@blocks/blocks'
 
 import * as themeComponents from '@theme-ui/components'
 

